content/reference/sql/server/backups.md

```diff
@@ -4,21 +4,91 @@ title: Backups
 
 # Backups
 
-Dolt has two options for backup: remotes and backups.
+Dolt has two options for [backups](../../../concepts/dolt/rdbms/backups.md): remotes and backups.
 
 ## Remotes
 
-Using remotes for backups should be suitable for some use cases. Pushing
-to a remote creates an off server copy of the branch being pushed.
-Frequently pushing to a remote can serve as a reasonable backup.
+Using remotes for backups should be suitable for some use cases. Using remotes for backups only backs up to the current commit of a branch, not the working set. Pushing to a remote creates an off server copy of the branch being pushed. Frequently pushing to a remote can serve as a reasonable backup.
+
+### Configure a remote
+
+Currently you can only add and remove remotes from the [Dolt CLI](../../cli.md). The example uses DoltHub as a remote but you can use Dolt with [other remotes like filesystem, AWS S3, and GCS](https://www.dolthub.com/blog/2021-07-19-remotes/). 
+
+```
```

Comment:
an assumption is made here that you have initialized an empty repository on Dolthub, `timsehn/backup-example`, and that you have configured credential access to that database (maybe link?)